xonsh/ansi_colors.py

```diff
@@ -1149,7 +1149,8 @@ def ansi_style_by_name(name):
     if name in ANSI_STYLES:
         return ANSI_STYLES[name]
     elif not HAS_PYGMENTS:
-        raise KeyError(f"could not find style {name!r}")
+        print(f"could not find style {name!r}, using 'default'")
```

Comment:
I think this kind of warnings should go to err stream.

Reply:
Yeah, I think you're right.  I can put in a follow up